src/server/templates/settings.html

```diff
@@ -1873,6 +1878,10 @@
 		<p><strong>{{ __('No configuration file was loaded. Falling back to default configuration.') | safe }}</strong></p>
 		<p>{{ __('Copy <em>server/config-dist.json</em> to <em>server/config.json</em> and then update settings for your server port number, admin username/password, and LED configuration.') | safe }}</p>
 	</div>
+{% elif note_message_text %}
+	<div class="config-none">
+		<p><strong>Note: </strong>{{ note_message_text | safe }}</p>
+	</div>
```

Comment:
The `config-none` class isn't really appropriate; we should define a semantic class for this purpose.

Reply:
I copied it from your code for showing the "No configuration file..." message. :]  Feel free to modify later on.

Reply:
We should address later, but no need to hold up the PR.